src/services/CertificateGenerationService.ts

```diff
@@ -244,7 +244,7 @@ class CertificateGenerationService {
    * @returns boolean true if Welsh
    */
   public async lookupPostcode(postcode: string): Promise<boolean> {
-      const axiosInstance = await axiosClient();
```

Comment:
Looks good to me, do you think it's worth adding a unit tests to prove that this moves onto the second attempt after 7000 ms?

Reply:
Not sure. I assume the 'if error retry' logic was tested originally, and really all I have done is prove axios will throw the error, which I would need to mock anyway.